readme.md

```diff
@@ -1,52 +1,27 @@
 # planetmintgo
-**planetmintgo** is a blockchain built using Cosmos SDK and Tendermint and created with [Ignite CLI](https://ignite.com/cli).
+**planetmintgo** is a blockchain built using Cosmos SDK and CometBFT and created with [Ignite CLI](https://ignite.com/cli).
```

Comment:
here i would just say `planetmint`, because that is the actual name.

Maybe: **planetmint** is a blockchain built using Cosmos SDK and CometBFT written in Go and created with [Ignite CLI](https://ignite.com/cli).